tests/Api/DomainsTest.php

```diff
@@ -0,0 +1,62 @@
+<?php
+
+/*
+ * Copyright (C) 2013-2016 Mailgun
+ *
+ * This software may be modified and distributed under the terms
+ * of the MIT license.  See the LICENSE file for details.
+ */
+
+namespace Mailgun\Tests\Api;
+
+/**
+ * @author Sean Johnson <sean@mailgun.com>
+ */
+class DomainsTest extends TestCase
+{
+    protected function getApiClass()
+    {
+        return 'Mailgun\Api\Domains';
+    }
+
+    /**
+     * NOTE: This test uses the production Mailgun API.
```

Comment:
Why? This should be avoided. If we need it we should group these tests and exclude them by default.